src/pages/home/report/PureReportActionItem.tsx

```diff
@@ -328,6 +328,18 @@ type PureReportActionItemProps = {
 const emptyHTML = <RenderHTML html="" />;
 const isEmptyHTML = <T extends React.JSX.Element>({props: {html}}: T): boolean => typeof html === 'string' && html.length === 0;
 
+const useNewTableReportViewActionRenderConditionals = ({childMoneyRequestCount, childVisibleActionCount, pendingAction, actionName}: OnyxTypes.ReportAction) => {
+    const previousChildMoneyRequestCount = usePrevious(childMoneyRequestCount);
+
+    return !(
+        actionName === CONST.REPORT.ACTIONS.TYPE.REPORT_PREVIEW &&
+        pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE &&
+        childMoneyRequestCount === 0 &&
+        (childVisibleActionCount ?? 0) > 0 &&
+        (previousChildMoneyRequestCount ?? 0) > 0
+    );
+};
```

Comment:
I think a method like this should be added to the utils file, and more docs should be added for better readability. Otherwise its harder for developers to follow what the method is for

Reply:
But that could be updated in a follow up

Reply:
+1 I found it confusing as someone who just looked at this for the first time. I would also say that it has a confusing name.

Reply:
Added it to the follow ups so we do not forget 🙌 https://github.com/Expensify/App/issues/59452